microsetta_interface/templates/source.jinja2

```diff
@@ -133,6 +136,9 @@
                 <div class="col-sm">
                     {{sample.sample_site |e}}
                 </div>
+                <div class="col-sm">
+                    <a class="btn btn-outline-info" href="/accounts/{{account_id}}/sources/{{source_id}}/samples/{{sample.sample_id}}/results">View</a>
```

Comment:
Is there a check to make sure there are results for the sample?

Reply:
Seemed to have overlooked exposing and endpoint for checking if sample results exist in `microsetta_public_api`. Can wrap it in with a new feature. Something like `/results-api/diversity/alpha/exists?sample_id=some_id`

Reply:
And likely a similar feature for taxonomy

Reply:
okay. What happens if a result has an alpha diversity value but not a taxonomy value? (shouldn't happen, but technically a possible outcome)

Reply:
https://github.com/biocore/microsetta-interface/pull/11/commits/f00f8c492500290f2a0afdc5b6734cc39bf37b63
Only populates the button if the data have existing alpha diversity and taxonomy
![Screen Shot 2020-07-27 at 9 52 32 PM](https://user-images.githubusercontent.com/19470970/88620876-eb7e3000-d053-11ea-8d25-653a93935b4d.png)